### DISCUSSION TITLE
Uploading Document For Github Education Profile Activation Not Working

### DISCUSSION THREAD

**Discussion by maahi-kaur-stu** [community/community#88636] [GitHub Education] [answered]
### Hi everyone,

Hi Everyone, 
While Uploading certain document to Authenticate myself as a Student for Github Education API is failing.
![Screenshot 2024-01-18 at 9 42 54 AM](https://github.com/community/community/assets/147657140/cf05bee6-fd63-440a-8514-bb92bfae2c4b)

Can anyone look into this and revert me back?

**Comment by mishmanners** [comment, marked as answer]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:
- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB
